tests/main/protocol-spec.ts

```diff
@@ -134,18 +134,18 @@ describe('protocol', () => {
     });
 
     it('handles an electron path url', () => {
-      // electron-fiddle://electron/{ref}/{path}
+      // electron-fiddle://electron/{tag}/{path}
       listenForProtocolHandler();
 
       const handler = (app.on as any).mock.calls[0][1];
 
-      handler({}, 'electron-fiddle://electron/4.0.0/test/path');
+      handler({}, 'electron-fiddle://electron/v4.0.0/test/path');
```

Comment:
I'm not blocking on this, more thinking out loud - is there any other areas where this `v` prefix may cause issues? Trying to decide if we should just strip this off of the tag name when we fetch and pass through the tag version.

Reply:
I don't think it will cause any problems - this was actually a case of the tests not reflecting reality, since AFAIK the URL has always included the `v` prefix in practice. In fact if you strip it off and send just the version number without the prefix, the current Fiddle won't work with that URL (see screenshot) since there's no ref on the `e/e` repo for that name. So the tests were covering a case which didn't actually work in practice, since the `octo` response was mocked.

<img width="1512" alt="Screen Shot 2022-12-07 at 1 18 02 PM" src="https://user-images.githubusercontent.com/5820654/206298419-6d72d9b8-41d6-43e0-8e9d-b7768bcd6f62.png">

The logic here before was generic enough that you could use URLs like `electron-fiddle://electron/v20.0.0/`, `electron-fiddle://electron/20-x-y/`, or `electron-fiddle://electron/f20a51a/`. This just drops support for the latter two options since it's no longer possible to easily get a concrete version number from `20-x-y` or `f20a51a`.